app/assets/stylesheets/styles.scss

```diff
@@ -22,6 +21,15 @@ $bandtree-green: #43ac6a;
   text-align: center;
 }
 
+.splash-text {
+  background-color: #222222;
+  font-size: 2em;
+  line-height: 36px;
+  margin: 0 -15px 0 -15px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 -15px` instead of `0 -15px 0 -15px`